src/Integration.Vsix.UnitTests/InfoBar/InfoBarManagerTests.cs

```diff
@@ -114,6 +117,7 @@ public void InfoBarManager_AttachInfoBar()
             infoBarUI.Model.TextSpans.GetSpan(0).Text.Should().Be("Hello");
             infoBarUI.Model.ActionItems.Count.Should().Be(1);
             infoBarUI.Model.ActionItems.GetItem(0).Text.Should().Be("world");
+            infoBarUI.Model.ActionItems.GetItem(0).IsButton.Should().BeTrue();
```

Comment:
The old method should create infobars with button style

Reply:
Perhaps call this out in a comment in the manager class? It's not obvious that it's being done to preserve the legacy behaviour.

Reply:
The comment is on the interface itself. WDYT?